shifthelper/db_cloner/__main__.py

```diff
@@ -52,12 +47,27 @@ def users():
 
 
 def main():
+    log = logging.getLogger()
+    log.setLevel('INFO')
+    logfile = os.environ.get(
+        'SHIFTHELPER_DBCLONER_LOG',
+        os.path.join(os.environ['HOME'], '.shifthelper', 'dbcloner.log')
+    )
+    formatter = logging.Formatter(
+        fmt='%(asctime)s|%(name)s|%(levelname)s|%(module)s|%(lineno)d|%(message)s'
+    )
+    formatter.converter = time.gmtime  # use utc in log
+    for handler in (logging.FileHandler(logfile), logging.StreamHandler()):
+        handler.level = logging.INFO
+        handler.setFormatter(formatter)
+        log.addHandler(handler)
+
```

Comment:
These new environment vars, which the user/co-developer can use to steer stuff should also be mentioned in the README, I guess.